Makefile

```diff
@@ -15,6 +15,12 @@ android_framework:
 	gomobile bind -target=android -o textilego.aar github.com/textileio/textile-go/mobile github.com/textileio/textile-go/net
 	cp -r textilego.aar ~/github/textileio/textile-mobile/android/textilego/
 
+P_TIMESTAMP = Mgoogle/protobuf/timestamp.proto=github.com/golang/protobuf/ptypes/timestamp
+P_ANY = Mgoogle/protobuf/any.proto=github.com/golang/protobuf/ptypes/any
+PKGMAP = $(P_TIMESTAMP),$(P_ANY)
+protos:
+	cd pb/protos && PATH=$(PATH):$(GOPATH)/bin protoc --go_out=$(PKGMAP):.. *.proto
```

Comment:
now using protobuffs instead of JWTs over the wire... was already part of the OB net setup, so went with it